docs/usage.md

```diff
@@ -6,61 +6,162 @@
+The `oncoanalyser` pipeline typically runs from FASTQs or BAMs and supports two modes: (1) whole genome and/or
+transcriptome, and (2) targeted panel. Launching an analysis requires only the creation of a samplesheet that describes
+details of each input such as the sample type (tumor or normal), sequence type (DNA or RNA), and filepath.
 
-## Samplesheet input
+Various aspects of an `oncoanalyser` analysis can be configured to fit a range of needs, and many of these are
+considered [advanced usage](#advanced-usage) of the pipeline. The most useful include:
 
-You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.
+- precise process selection
+- starting from existing data
+- granular control over reference/resource files
 
-```bash
---input '[path to samplesheet file]'
+These features enable `oncoanalyser` to be run in a highly flexible way. For example, an analysis can be run with
+existing PURPLE data as the starting point and skip variant calling processes. Additionally, reference/resource files
+can be staged locally to optimise execution or modified to create user-defined driver gene panels.
+
+:::danger
+
+When starting from BAMs rather than FASTQ it is expected that:
+
+- RNA read alignments are generated with STAR using [specific
+  parameters](https://github.com/hartwigmedical/hmftools/tree/master/isofox#a-note-on-alignment-and-multi-mapping), this
+  is **critical** for WTS data, and
+- reads are aligned to a Hartwig-distributed reference genome ([custom genomes](#custom-genomes) can be used but are not
+  recommended)
+
+:::
+
+## Supported analyses
+
+A variety of analyses are accessible in `oncoanalyser` and are implicitly run according to the data described in the
+samplesheet. The supported analysis types for each workflow are listed below.
+
+| Input sequence data                 |  WGS/WTS workflow  | Targeted sequencing workflow<sup>\*</sup> |
+| ----------------------------------- | :----------------: | :---------------------------------------: |
+| • Tumor/normal DNA<br />• Tumor RNA | :white_check_mark: |                     -                     |
+| • Tumor only DNA<br />• Tumor RNA   | :white_check_mark: |            :white_check_mark:             |
+| • Tumor/normal DNA                  | :white_check_mark: |                     -                     |
+| • Tumor only DNA                    | :white_check_mark: |            :white_check_mark:             |
+| • Tumor only RNA                    | :white_check_mark: |                     -                     |
+
+<sub><sup>\*</sup> Supported analyses relate to the TSO500 panel only</sub>
+
+## Samplesheet
```

Comment:
Can you note somewhere why you have double underscores and what happens with those?

Reply:
I have to apologise for the misunderstand on this one - the double underscores are only used in the example samplesheets to improve readability for users; they are not considered internally in any way by oncoanalyser.

Reply:
- Agreed, double underscores removed in https://github.com/nf-core/oncoanalyser/pull/68